src/App.js

```diff
@@ -1,5 +1,25 @@
-import React from "react";
+import React, { useEffect } from "react";
+import { Routes, Route } from "react-router-dom";
+import { useVideosDataProvider } from "./Context/VideosDataContext/VideosDataProvider";
+
+import { Home, Navbar } from "./Components";
+
+import { getVideos } from "./utilities/backendRequest";
+
+import "./styles.css";
```

Comment:
I will suggest removing extra space

Reply:
I would say batch your imports into logical sections 

Like 

import from the dependencies 
--space--
import from the src files ( components and other function) 
--space--
import CSS files